### PR TITLE
ADEN-7180 Adds tracking opt in context

### PR DESCRIPTION
**Pull request by leshniak**
## Links

* http://wikia-inc.atlassian.net/browse/ADEN-7180

@fraszczakszymon

### CODE REVIEW

**Review by nandy-andy — APPROVED** [review]

**Review by hakubo — CHANGES_REQUESTED** [review]
Please use npm 6.0.1 or newer

**Review by fraszczakszymon — APPROVED** [review]

**Comment by leshniak** [comment]
@hakubo done.

File: app/modules/ads/setup.js [hunk 1]
```diff
@@ -78,10 +79,12 @@ function setupAdContext(adsContext, instantGlobals) {
 	slots.setupStates();
 }
 
-function configure(adsContext, instantGlobals) {
+function configure(adsContext, instantGlobals, isOptedIn) {
 	const {context} = window.Wikia.adEngine;
+	const {utils: adProductsUtils} = window.Wikia.adProducts;
```
Comment (hakubo) [inline app/modules/ads/setup.js:84]: const adProductsUtils = window.Wikia.adProducts.utils;
looks more readable IMO

**Comment by hakubo** [comment]
it looks like you did not commit package-lock with latest comment

File: app/modules/ads/setup.js (hunk 1 above)
Comment (leshniak) [reply app/modules/ads/setup.js:84]: I agree, but this convention allows us to switch to imports easily – it's a temporary solution. I mean everything in the destructuring statement is an import from "adProducts".

**Comment by leshniak** [comment]
@hakubo yup, we have a problem with our jenkins job. Updated manually.

**Review by fraszczakszymon — APPROVED** [review]

**Review by hakubo — APPROVED** [review]

**Pull request merged**